crates/lib/src/address.rs

```diff
@@ -23,6 +23,12 @@ pub struct Address {
 
 impl_st!([] Address, _ts => AlgebraicType::product([("__address_bytes", AlgebraicType::bytes())]));
 
+impl Default for Address {
```

Comment:
Maybe a comment about what, if any, semantics that an all-zeroes address is imbued with. This might be documented at the point that Address is defined.